resources/services/glue/security_configurations.hcl

```diff
@@ -0,0 +1,42 @@
+service          = "aws"
+output_directory = "."
+add_generate     = true
+
+
+description_modifier "remove_read_only" {
+  words = ["  This member is required."]
+}
+
+resource "aws" "glue" "security_configurations" {
+  path = "github.com/aws/aws-sdk-go-v2/service/glue/types.SecurityConfiguration"
+  ignoreError "IgnoreAccessDenied" {
+    path = "github.com/cloudquery/cq-provider-aws/client.IgnoreAccessDeniedServiceDisabled"
+  }
+  deleteFilter "AccountRegionFilter" {
+    path = "github.com/cloudquery/cq-provider-aws/client.DeleteAccountRegionFilter"
+  }
+  multiplex "AwsAccountRegion" {
+    path   = "github.com/cloudquery/cq-provider-aws/client.ServiceAccountRegionMultiplexer"
+    params = ["glue"]
+  }
+  options {
+    primary_keys = ["account_id", "name"]
+  }
+  userDefinedColumn "account_id" {
+    description = "The AWS Account ID of the resource."
```

Comment:
```suggestion
    description = "The AWS Account ID of the resource"
```

Reply:
@amanenk This is totally a nitpick, but it looks like it wasn't fixed in the most recent update